netlib/http/headers.py

```diff
@@ -0,0 +1,205 @@
+"""
+
+Unicode Handling
+----------------
+See also: http://lucumr.pocoo.org/2013/7/2/the-updated-guide-to-unicode/
+"""
+from __future__ import absolute_import, print_function, division
+import copy
+try:
+    from collections.abc import MutableMapping
+except ImportError:  # Workaround for Python < 3.3
+    from collections import MutableMapping
```

Comment:
related to https://github.com/python-hyper/hyperframe/commit/49f94ee5e3fb5b2002c6e465b0c717ce402822ab ?